client/post-editor/editor-revisions/dialog.jsx

```diff
@@ -79,7 +79,11 @@ class PostRevisionsDialog extends PureComponent {
 	dialogButtons = () => {
 		const { postId, revision, siteId, translate } = this.props;
 		return [
-			{ action: 'cancel', compact: true, label: translate( 'Cancel' ) },
+			{
+				action: 'cancel',
+				compact: true,
+				label: translate( 'Cancel', { context: 'modal button' } ),
```

Comment:
:disappointed: 29 existing translations will be lost with this change.
If the string must change, the following option has already been translated **28** times:
`translate( 'Cancel', { context: 'Customizer cancel element editing button'} )` *ES Score: 14*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/post-revisions/add-context-to-ambiguous-translations)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).